Assets/LeapMotion/Core/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -547,6 +605,21 @@ public class RuntimeGizmoDrawer {
       PopMatrix();
     }
 
+    /// <summary>
+    /// Draws a wire gizmo circle at the given position, with the given radius;
+    /// the normal is assumed to be towards Camera.main.
+    /// </summary>
+    public void DrawWireCircle(Vector3 center, float radius) {
```

Comment:
The current DrawWireCircle that accepts a normal value will include a cross in the center (since it is a squashed wire sphere actually).  Since the method names are exactly the same, they should probably have the same appearance.  I'd recommend changing the existing one to use DrawWireArc as well.

(and this method should probably just calculate the circle normal and then use the existing DrawWireCircle)

Reply:
Will look at adding code re-use, but I think DrawWireCircle should never draw a cross in the center because that's not what a wire circle is (this actually made DrawWireCircle unusable for me)

Reply:
Makes sense, in that case, maybe the original DrawWireCircle needs to be updated to use the new arc code?  That way everything is consistent.

Reply:
Can do :+1: